eras/conway/impl/src/Cardano/Ledger/Conway/Governance/Procedures.hs

```diff
@@ -565,8 +567,8 @@ instance EraPParams era => EncCBOR (GovAction era) where
         Sum TreasuryWithdrawals 2 !> To ws
       NoConfidence gid ->
         Sum NoConfidence 3 !> E (encodeNullStrictMaybe encCBOR) gid
-      NewCommittee gid old new ->
-        Sum NewCommittee 4 !> E (encodeNullStrictMaybe encCBOR) gid !> To old !> To new
+      NewCommittee gid old new q ->
```

Comment:
Could we change the name as well? to `UpdateCommitte`, since we're changing the cddl anyway?  Would that cause too big discrepancies with what is already integrated?

Reply:
I think it is a great idea. Please, go ahead, change it in the cddl spec as well as Haskell code. This will reflect better the actual semantics that are described in the CIP-1694

Reply:
I changed it without deprecation or anything, I suppose they will have to reset sanchonet if this action was already submitted, but I assume that's find at this point

Reply:
Yes, we are allowed to break SanchoNet :grin:

Reply:
:fire: